pyCardDeck/deck.py

```diff
@@ -263,6 +266,77 @@ def show_top(self, number: int) -> List[CardType]:
         """
         return self._cards[0:number]
 
+    def set_file_location(self, location):
+        """
+        used to update the location
+        it will expand relative file path and ~ if it exists
+
+        :param location: the location of the file
```

Comment:
Yes relative to the main script
It has been assumed the file name is passed it but it will be documented